PC02_Graffiti_Bennett.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Laurel Bennett
```

Comment:
I think this code was really funny and ran without any bugs. My largest criticism is that Laurel could have utilized the goto() function more which would have made the code more D.R.Y. Prior to editing, the drawings were very funny. It also highlighted Laurel's ability to adapt a command she already knew like circle() and manipulate it to do something entirely different.